addons/web/static/tests/views/calendar/calendar_view_tests.js

```diff
@@ -4980,4 +4982,54 @@ QUnit.module("Views", ({ beforeEach }) => {
             assert.containsOnce(target, ".o_view_sample_data", "should have sample data");
         }
     );
+
+    QUnit.test(`Resizing Pill of Multiple Days(Allday)`, async (assert) => {
+        const { advanceTime } = mockTimeout();
+        await makeView({
+            type: "calendar",
+            resModel: "event",
+            serverData,
+            arch: `
+                <calendar event_open_popup="1" date_start="start" date_stop="stop" all_day="allday" delete="0" mode="month" >
+                    <field name="stop"/>
+                </calendar>`,
+            mockRPC(route, { args, method }) {
+                if (method === "create") {
+                    assert.deepEqual(
+                        args[0],
+                        [
+                            {
+                                allday:true,
+                                name: "new event",
+                                start: "2016-12-25",
+                                stop: "2016-12-28",
+                            },
+                        ],
+                        "should send the correct data to create events"
+                    );
+                }
+                if (method === "write") {
+                    assert.deepEqual(args[1], {
+                        allday: true,
+                        start: "2016-12-25",
+                        stop: "2016-12-31",
+                    });
+                }
+            },
+        });
+
+        await selectAllDayRange(target, "2016-12-25", "2016-12-28");
+        await editInput(target, ".o-calendar-quick-create--input", "new event");
+        await click(target, ".o-calendar-quick-create--create-btn");
+        await resizeEventToDate(target, 8, "2016-12-31");
+        await clickEvent(target, 8);
+        await advanceTime(300);
+        assert.strictEqual(
+            target.querySelector(
+                ".o_cw_popover .o_cw_popover_fields_secondary .list-group-item .o_field_datetime"
+                ).textContent.split(' ')[0],
```

Comment:
According to the web tooling (https://github.com/odoo/odoo/blob/62cbb2073922db98df380eacce0b620e1dda9877/addons/web/tooling/enable.sh). You can install that tooling by executing that file in your terminal.

```suggestion
                ).textContent.split(" ")[0],
```